roles/network_plugin/contiv/templates/contiv-config.yml.j2

```diff
@@ -0,0 +1,29 @@
+# This ConfigMap is used to configure a self-hosted Contiv installation.
+# It can be used with an external cluster store(etcd or consul) or used
+# with the etcd instance being installed as contiv-etcd
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: contiv-config
+  namespace: {{ system_namespace }}
+data:
+  # The location of your cluster store. This is set to the
+  # avdertise-client value below from the contiv-etcd service.
+  # Change it to an external etcd/consul instance if required.
+  cluster_store: "etcd://127.0.0.1:6666"
```

Comment:
Can you make the contiv_etcd_port configurable? Just in case 6666 is not available on the host.